docs/source/configuration/authorization.mdx

```diff
@@ -0,0 +1,596 @@
+
+  ```mermaid
+  flowchart LR;
+    clients(Client);
+    subgraph Router[" "]
+      router(["<b>Apollo Router</b>"]);
+      serviceB[Users<br/>API];
+      serviceC[Posts<br/>API];
+    end
+    router -->|"✅ Authorized <br/> subquery"| serviceC;
+      router -.->|"❌ Unauthorized <br/> subquery"| serviceB;
+    clients -->|"⚠️ Partially authorized <br/>request"| router;
+  ```
+  - [Query deduplication](./traffic-shaping/#query-deduplication) also accounts for authorization by grouping requested fields based on their required authorization. Entire groups can be eliminated from the query plan if they don't have the correct authorization.
+
+- **Declarative access rules**: You define access controls at the field level, and GraphOS [composes](#composition-and-federation) them across your services. These rules create graph-native governance without the need for an extra orchestration layer.
+
+- **Principled architecture**: Through composition, the router centralizes authorization logic while allowing for auditing at the service level. This centralized authorization is an initial checkpoint that other service layers can reinforce.
+
+  ```mermaid
+  flowchart LR;
+    clients(Clients);
+    Level2:::padding
+    subgraph Level1["<br>🔐 Router layer&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp"]
+      router(["<b>Apollo Router</b>"]);
+        subgraph Level2["🔐 Service layer"]
+          serviceB[Users<br/>API];
+          serviceC[Posts<br/>API];
+          end
+    end
+
+    router -->|"Subquery"| serviceB & serviceC;
+    clients -->|"Request"| router;
+
+  classDef padding padding-left:1em, padding-right:1em
+  ```
+
+## How it works
+
+The Apollo Router provides access controls via **authorization directives**. Using the `@requiresScopes` and `@authenticated` directives, you can define access to specific fields and types across your supergraph:
+
+- The [`@requiresScopes`](#requiresscopes) directive allows granular access control through the scopes you define.
+- The [`@authenticated`](#authenticated) directive works in a binary way: authenticated requests can access the specified field or type, and unauthenticated requests can't.
+
+For example, imagine you're building a social media platform that includes a `Users` subgraph. You can use the [`@requiresScopes`](#requiresscopes) directive to declare that viewing other users' information requires the `read:user` scope:
+
+```graphql
+type Query {
+  users: [User!]! @requiresScopes(scopes: ["read:users"])
```

Comment:
```suggestion
  users: [User!]! @requiresScopes(scopes: [["read:users"]])
```

Input coercion for lists cannot transform from `[Type]` to `[[Type]]` https://spec.graphql.org/June2018/#sec-Type-System.List 

Currently federation would still accept to compose this though. More context in https://github.com/apollographql/federation/issues/2744